sections/footer.liquid

```diff
@@ -73,14 +73,36 @@
                       {%- endfor -%}
                     </ul>
                   {%- endif -%}
+                {%- when 'brand_information' -%}
+                  <div class="footer-block-brand-information">
+                    {%- if settings.brand_image != blank -%}
+                      {%- assign image_size_2x = settings.brand_image.width | times: 2 | at_most: 5760 -%}
```

Comment:
I think here, since we know what the max value can be, `550px` then we should use that. Otherwise if a merchant use a large image we're going to be loading an image super heavy without any real benefit but a hit on performance. 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/02-26-r7guo-0nluw.png)
</details>

And when it uses `at_most: 550` then the weight is 10 times lighter and still look good 👍